addons/gamification/data/gamification_karma_rank_data.xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo><data noupdate="1">
     <!--Karma-->
-    <record id="base.user_root" model="res.users">
-        <field name="karma">2500</field>
-    </record>
-    <record id="base.user_admin" model="res.users">
-        <field name="karma">2500</field>
+    <record id="karma_tracking_user_root_0" model="gamification.karma.tracking">
```

Comment:
The commit message can be a little more complete, e.g. you never say that the old_value is now computed based on the new_value of the previous record, and what it implies (readonly to avoid inconsistency, etc)